lint-tasks.yml

```diff
@@ -1,20 +1,43 @@
 version: "3"
 
 vars:
+  G_SRC_CLP_FFI_JS_DIR: "{{.ROOT_DIR}}/src/clp_ffi_js"
   G_LINT_VENV_DIR: "{{.G_BUILD_DIR}}/lint-venv"
 
 tasks:
   check:
     cmds:
+      - task: "cpp-check"
       - task: "yml-check"
 
   fix:
     cmds:
+      - task: "cpp-fix"
       - task: "yml-fix"
 
   cpp-configs:
     cmd: "{{.ROOT_DIR}}/tools/yscope-dev-utils/lint-configs/symlink-cpp-lint-configs.sh"
 
+  cpp-check:
+    sources: &cpp_source_files
+      - "{{.G_SRC_CLP_FFI_JS_DIR}}/.clang-format"
+      - "{{.G_SRC_CLP_FFI_JS_DIR}}/**/*.cpp"
+      - "{{.G_SRC_CLP_FFI_JS_DIR}}/**/*.h"
```

Comment:
Just curious since I'm not too familiar with our C++ code conventions, if we're not sticking with `*.hpp` for headers, when do we use `*.h`?

Reply:
`.hpp` are meant to be C++ headers. `.h` are meant to be C-compatible headers.

Reply:
Right, are there cases we need our own C-compatible headers in `src/clp-ffi-js`?

Reply:
Not that I can think of, but this is more of a default setting in case we have them one day. I could see us forgetting about it and then the file doesn't get formatted properly, lol.